### ISSUE TITLE
Mongoid initial value not setted

### ISSUE THREAD

**Issue by nofxx** [aasm/aasm#206]
Won't work.

```ruby
class Note
  include Mongoid::Document
  include Mongoid::Timestamps
  include AASM
    
  field :aasm_state, type: String
  aasm do
    state :queued, initial: true
  end
end

Note.new.aasm_state # => nil
Note.new.queued? # => nil
```

Tryed custom column: 'state' with same error.

Workaround:  `field :aasm_state, type: String, default: :queued`

PS.: Thank you for AASM, hard to find, bad searches engines...the plural... a SEO improvement might be having the singular 'state machine' somewhere.

**Comment by nofxx** [comment]
Willing to accept a patch that creates the field automagically?

**Comment by alto** [comment]
@nofxx Absolutely! I'm happy to accept patches (with tests).

**Comment by alto** [comment]
Thanks to @Shwetakale this has been fixed. :+1: 

I will release a new version soon.

**Comment by alto** [comment]
I just released version `4.1.0` including the changes.

**Comment by Shwetakale** [comment]
@alto Thanks :)
